apps/condo/domains/common/components/PostMessageProvider/globalHandlers.ts

```diff
@@ -0,0 +1,138 @@
+import { useCallback, useContext } from 'react'
+import { notification } from 'antd'
+import { v4 as uuidV4 } from 'uuid'
+import get from 'lodash/get'
+import pickBy from 'lodash/pickBy'
+import dayjs from 'dayjs'
+import { useIntl } from '@open-condo/next/intl'
+import { useAuth } from '@open-condo/next/auth'
+import { useOrganization } from '@open-condo/next/organization'
+import { STAFF } from '@condo/domains/user/constants/common'
+import { useMiniappTaskUIInterface } from '@condo/domains/common/hooks/useMiniappTaskUIInterface'
+import { TASK_STATUS, TasksContext } from '@condo/domains/common/components/tasks'
+import type { RequestHandler } from './types'
+
+export const handleNotification: RequestHandler<'CondoWebAppShowNotification'> = (params) => {
+    const { type, ...restParams } = params
+    notification[type](restParams)
+    return { success: true }
```

Comment:
Always successful result? Is there an ability to fail? What if `notification[type]` is not defined?

Reply:
Please see handle logic...

We use ajv to validate params....so there (inside handler) will always be correct params (in terms of Bridge API) ['info', 'success', 'error', 'warning']. Even if not and this part failed somehow it will throw error. 

All event handlers are wrapped in `try...catch` block, so it will catch error and send error event with type `HANDLER_ERROR`